samples/snippets/batch_process_documents_sample_v1beta3_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+from uuid import uuid4
+
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+import pytest
+
+from samples.snippets import batch_process_documents_sample_v1beta3
+
+project_id = "python-docs-samples-tests"
+location = "us"
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+processor_id = "90484cfdedb024f6"
+gcs_input_uri = "gs://cloud-samples-data/documentai/invoice.pdf"
+gcs_output_uri = "gs://document-ai-python"
```

Comment:
Is there a reason why the output is going to a static bucket instead of just creating a bucket that you delete at the end?

Reply:
I used the same process as the Node samples, but I'll ask Eric about it. Now that you say it, it does make sense to just tear it down at the end.

Reply:
I'm back! I was deleting the files in the bucket, but forgot to delete the bucket itself. Resolved!